integration/indexes_compat_test.go

```diff
@@ -451,27 +455,31 @@ func TestCreateIndexesCommandCompat(t *testing.T) {
 
 			assert.Equal(t, compatRes, targetRes)
 
-			targetErr = targetCollection.Database().RunCommand(
-				ctx, bson.D{{"listIndexes", tc.collectionName}},
-			).Decode(&targetRes)
+			targetCursor, targetErr := targetCollection.Indexes().List(ctx)
+			compatCursor, compatErr := compatCollection.Indexes().List(ctx)
 
-			compatErr = compatCollection.Database().RunCommand(
-				ctx, bson.D{{"listIndexes", tc.collectionName}},
-			).Decode(&targetRes)
+			if targetCursor != nil {
+				defer targetCursor.Close(ctx)
+			}
+			if compatCursor != nil {
+				defer compatCursor.Close(ctx)
+			}
 
-			require.Nil(t, targetRes)
-			require.Nil(t, compatRes)
+			require.NoError(t, targetErr)
+			require.NoError(t, compatErr)
 
-			if targetErr != nil {
-				t.Logf("Target error: %v", targetErr)
-				t.Logf("Compat error: %v", compatErr)
+			targetListRes := FetchAll(t, ctx, targetCursor)
```

Comment:
We use fetchall because listIndexes run command returns cursor. It's easier to test with `FetchAll` for now.